File: db4on/core/src/core/com/db4o/DeleteInfo.cs
namespace com.db4o
{
	public class DeleteInfo : com.db4o.TreeInt
	{
		public bool _delete;

		internal int _cascade;

		public com.db4o.YapObject _reference;

		public DeleteInfo(int id, com.db4o.YapObject reference, bool delete, int cascade)
			 : base(id)
		{
			_reference = reference;
			_delete = delete;
			_cascade = cascade;
		}

		public override object ShallowClone()
		{
			com.db4o.DeleteInfo deleteinfo = new com.db4o.DeleteInfo(0, _reference, _delete, 
				_cascade);
			return ShallowCloneInternal(deleteinfo);
		}
	}
}

File: db4oj/core/src/com/db4o/DeleteInfo.java
/* Copyright (C) 2004 - 2005  db4objects Inc.  http://www.db4o.com */

package com.db4o;


public class DeleteInfo extends TreeInt{
    
    public boolean _delete;
    int _cascade;
    public YapObject _reference;

    public DeleteInfo(int id, YapObject reference, boolean delete, int cascade) {
        super(id);
        _reference = reference;
        _delete = delete;
        _cascade = cascade;
    }
    public Object shallowClone() {
    	DeleteInfo deleteinfo= new DeleteInfo(0,_reference, _delete, _cascade);
    	return shallowCloneInternal(deleteinfo);
    }
    

}
